uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/sickness-leave-request-instance.js

```diff
@@ -18,42 +18,24 @@ define([
          * @return {object}
          */
         defaultCustomData: function () {
-          return {
-            reason: null,
-            required_documents: ''
-          }
-        },
+          var sicknessCustomData = {
+            sickness_reason: null,
+            sickness_required_documents: '',
+            request_type: 'sickness'
+          };
 
-        /**
-         * Create a new sickness request
-         *
-         * @return {Promise} Resolved with {Object} Created Leave request with
-         *  newly created id for this instance
-         */
-        create: function () {
-          return LeaveRequestAPI.create(this.toAPI(), 'sick')
-            .then(function (result) {
-              this.id = result.id;
-            }.bind(this));
+          return _.assign({}, LeaveRequestInstance.defaultCustomData(), sicknessCustomData);
         },
 
         /**
-         * Validate sickness request instance attributes.
+         * Gets array of documnets from comma separated string of documents
          *
-         * @return {Promise} empty array if no error found otherwise an object
-         *  with is_error set and array of errors
+         * @return {Array}
          */
-        isValid: function () {
-          return LeaveRequestAPI.isValid(this.toAPI(), 'sick');
-        },
+        getDocumentArray: function () {
```

Comment:
This function seems to be private as it is not used outside of the class, then please make it a private function instead of exposing it.